CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Helpers/ImageMetaInformationManager.swift

```diff
@@ -231,16 +232,29 @@ final class ImageMetaInformationManager {
         return nil
     }
 
+    fileprivate func entryFieldString(_ entryPoint: GiniConfiguration.GiniEntryPoint) -> String {
+            var stringValue: String
```

Comment:
[suggestion]: There is any reason why we can not do it directly in GiniConfiguration something like this:
```
public enum GiniEntryPoint: String {
        case button
        case field
    }
```
instead of declaring the enum of type Int and having this extra method for getting the string value of each case.

Reply:
I was also thinking about this for Android. I decided to do it like Nadya and separated enum values from their exif string encoding. If we use the enum value names directly in the exif, then if we change the enum values we also directly affect the exif and this would also not be obvious in the code.

If we keep enum values and their exif string encoding separate we avoid accidentally changing what we write into the exif. This is especially important because the analytics requies specific values in the exif.

Reply:
But on iOS we can do like this: 
```
public enum GiniEntryPoint: String {
        case button = "button"
        case field = "field" 
    }
```
or we can have a computed property for the `stringValue` like this:
```
public enum GiniEntryPoint {
        case button
        case field

var stringExifValue: String {
      switch self {
            case .button:
                return "button"
            case .field:
                return "field"
            }
      } 
    }
```
Any thoughts?

Reply:
@a-szotyori thanks for the comment :) you were very fast.

Reply:
@ValentinaIancu-Gini If the string value is important for exif, then I think it should stay in the class dealing with exif. In other words the string value of the enum is a concern for exif not for the enum therefore it should be closer to exif than to the enum.

Reply:
Ok, got it.
[suggestion]: we can rewrite the method to be more clear, efficient and short :
```
	fileprivate func entryFieldString(_ entryPoint: GiniConfiguration.GiniEntryPoint) -> String {
		switch entryPoint {
		case .button:
			return "button"
		case .field:
			return "field"
		}
	}
```